app/models/application.rb

```diff
@@ -0,0 +1,171 @@
+# == Schema Information
+#
+# Table name: applications
+#
+#  id                     :integer          not null, primary key
```

Comment:
I haven't really talked to you about this model so I'm probably missing something - each application doesn't have any indication of which event they're applying for?